ROLES.md

```diff
@@ -0,0 +1,110 @@
+# KServe Roles and Responsibilities
+
+This document outlines the various roles within the KServe community and their associated responsibilities.
+
+## Core Organization Roles
+
+### KServe Owners (`kserve-owners`)
+- Owners of the KServe GitHub organization
+- Have administrative access to all repositories
+- Responsible for high-level organization management
+- Can manage organization settings and permissions
+
+### Technical Steering Committee (`technical-steering-committee`)
+- Responsible for technical oversight of the KServe project
+- Makes decisions about project direction and technical matters
+- Reviews and approves major changes
+- Manages project resources and priorities
+- See [TECHNICAL-STEERING-COMMITTEE.md](./TECHNICAL-STEERING-COMMITTEE.md) for more details
+
+## Core Project Roles
+
+### KServe Committers (`kserve-committers`)
+- Have write access to repositories
+- Can approve and merge pull requests
+- Can triage and manage issues
+- Help maintain code quality and project standards
+- Mentor new contributors
+
+### KServe Reviewers (`kserve-reviewers`)
+- Listed in repository OWNERS files
+- Responsible for code review
+- Ensure code quality and project standards
+- Help maintain technical consistency
+- Can approve PRs but may not have merge rights
+
+### KServe Contributors (`kserve-contributors`)
+- All community members who contribute to KServe
+- Can submit pull requests
+- Can report and comment on issues
```

Comment:
This should be available for anyone. One special permission is that they can be assigned to issues